fuel-vm/src/gas/default_gas_costs.rs

```diff
@@ -77,6 +78,20 @@ pub fn default_gas_costs() -> GasCostsValues {
         time: 1,
         tr: 120,
         tro: 99,
+        wdcm: 2,
+        wqcm: 4,
+        wdop: 2,
+        wqop: 4,
+        wdml: 2,
+        wqml: 4,
+        wddv: 4,
+        wqdv: 8,
+        wdmd: 5,
+        wqmd: 10,
+        wdam: 2,
+        wqam: 4,
+        wdmm: 4,
+        wqmm: 8,
```

Comment:
Are these actually generated from our benchmarks?

Reply:
Those are from microbenchmarks. I should probably do a full benchmarking run using fuel-core gas benchmarking as well.

Reply:
Yeh they all need to be based on the same benchmark but I doubt it's a huge issue for math operations. We will have to re do all the gas soon anyway

Reply:
It will definitely be done before it reaches anywhere that would be accessible to the public

Reply:
I'm currently working on adding the instructions to fuel-core-benches. Do you think we should just merge this as-is and adjust the costs later?